ganga/GangaCore/Lib/Localhost/LocalHostExec.py.template

```diff
@@ -85,6 +88,27 @@ errorfile=open('stderr','w')
 sys.stdout=open('./__syslog__','w')
 sys.stderr=sys.stdout
 
+from Virtualization import checkDocker, checkUDocker, checkSingularity, installUdocker
+
+if (virtualization == "Docker"):
+    appscriptpath[0] = "/work_dir/"+appscriptpath[0]
+    if (checkDocker()):
+        print("Using Docker")
```

Comment:
Same kind of thought with the printing. If the default python version on your system is 2 then this will fail.

Reply:
Sorry I haven't used any versions before 2.7. So can you please explain why it won't work?

Reply:
I think this one is ok @mesmith75 Python 2.7+ supports the print function without the use of `__future__`